packages/bundler-security/tests/metroSecureResolver.test.ts

```diff
@@ -0,0 +1,75 @@
+import { checkSecurityViolation } from '../src/metroSecureResolver';
+
+describe('checkSecurityViolation', () => {
+    it('should detect violation when accessing @trezor scoped package from node_modules', () => {
```

Comment:
Its maybe a nit, but I think we are shall be using this format (I think I read it even somewhere in the Notion)

```suggestion
    it('detects violation when accessing @trezor scoped package from node_modules', () => {
```

Reply:
honestly I think it doesn't matter in way